charts/fluent-bit/Chart.yaml

```diff
@@ -5,7 +5,7 @@ keywords:
   - logging
   - fluent-bit
   - fluentd
-version: 0.31.0
+version: 0.32.0
```

Comment:
Should we finally do a 1.0.0 release at some point??

Reply:
I don't think this chart should ever hit v1.0.0, the new separate charts should be the focus.